modules/cms-kit/src/Volo.CmsKit.Public.Web/Pages/CmsKit/Shared/Components/Rating/default.js

```diff
@@ -0,0 +1,81 @@
+﻿(function () {
+    var l = abp.localization.getResource("CmsKit");
+    
+    $(document).ready(function () {
+        abp.widgets.CmsRating = function ($widget) {
+            var widgetManager = $widget.data("abp-widget-manager");
+            var $ratingArea = $widget.find(".cms-rating-area");
+            
+            function getFilters() {
+                return {
+                    entityType: $ratingArea.attr("data-entity-type"),
+                    entityId: $ratingArea.attr("data-entity-id")
+                };
+            }
+            
+            function registerCreateOfNewRating() {
+                var authenticated = $(".my-rating-5").attr("data-authenticated");
+
+                $(".my-rating-5").starRating({

```

Comment:
You should not query from global context. Use `$widget.find(".-my-rating-5")` to query inside this widget, to be modular and not effect the rest of the pages and other rating widgets.
Do also for others (like `.live-rating` below).